components/lib/crash/metrics.yaml

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the metrics that are recorded by glean telemetry. They are
+# automatically converted to Kotlin code at build time using the `glean_parser`
+# PyPI package.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+crash_metrics:
+  uncaught_exception:
+    type: counter
+    description: >
+      Counts the number of uncaught exceptions raised by the application.
+    bugs:
+      - 1522461
+    data_reviews:
+      - TBD
+    notification_emails:
+      - glean-team@mozilla.com
```

Comment:
I'm not sure this should be us. @georgf , thoughts?

Reply:
We are doing the initial work, but we shouldn't own this.
Sebastian or Stefan would probably know which email should go there.

Reply:
Sounds good to me.

@pocmo or @st3fan, would you mind helping to answer our question about who should/will own crash counts in lib-crash?  Basically we just need a point of contact email to insert here for ownership purposes.  Thanks!

Reply:
For now that's probably `android-components-team@mozilla.com` - does it make sense to add both teams AC+Glean?

Reply:
I think we should only add actual owners of the metric to this field.
While we're helping out with this, our team wouldn't be the owner of it going forward.